src/unit-test/Recipe.test.js

```diff
@@ -15,3 +15,11 @@ it('renders the recipe image', () => {
   const wrapper = shallow(<Recipe image={imgUrl}/>);
   expect(wrapper.find("img").prop("src")).toEqual(imgUrl)
 });
+
+it('changes state of showRecipeExpanded', () => {
```

Comment:
I am increasingly confused but what we should test for. Rule of thumb is behaviour not state. However, in React state means something slightly different - right?
This test is expressly testing for state - is it needed?
What happens if you remove? Does test coverage disappear?

Reply:
I've changed the name as it was misleading. What I test for is the behaviour of handleClick. I check state prior to calling the function and then after to make sure the behavior of the function is correct. It's my understanding that this would be considered checking behavior but correct me if I'm wrong.